### PR TITLE
Add italian language for recommended.md and resources.md

### PR DESCRIPTION
**Pull request by alessiobenvenuti**

### CODE REVIEW

File: recommended-it.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+## API testing
+
+
+- [Insomnia](https://insomnia.rest/)
+	- Scaricabile
+	- Esegui richieste API dall'applicazione.
+- [Postman](https://www.postman.com/)
+	- Scaricabile
+
```
Comment (MichaelCurrin) [inline recommended-it.md:97]:
```suggestion
```

File: resources-it.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# 📚 Risorse
+> Cenni su alcune idee chiave per gli sviluppatori con collegamenti esterni per approfondire
```
Comment (MichaelCurrin) [inline resources-it.md:2]:
```suggestion
> Cenni su alcune idee chiave per gli sviluppatori con collegamenti esterni per approfondire

```

File: resources-it.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Questa guida presuppone che tu abbia già delle conoscenze di programmazione e desideri fare qualcosa come:
+
+- esplorare alcuni argomenti di cui hai sentito parlare ma di cui non sai molto. O
+- imparare la tecnologia in maniera pratica e vuoi trovare alcune definizioni e spiegazioni più formali. O,
+- scoprire cosa c'è da sapere e come queste cose si incastrano tra di loro.
+
+Nella mia esperienza nei colloqui(lavorativi) di ingegneria del software (sia facendo dei colloqui sia affrontando dei colloqui), questi sono i tipi di argomenti che emergono molto e quindi conoscerli almeno a livello concettuale ti aiuterà molto.
+
+Non è necessario conoscere effettivamente tutti questi argomenti, ma idealmente dovresti sapere quali sono la maggior parte di questi elementi e quando e perché li utilizzeresti. Laddove i collegamenti non sono forniti di seguito, è possibile cercarli da soli su Wikipedia, blog o tutorial.
+
```
Comment (MichaelCurrin) [inline resources-it.md:14]:
```suggestion


```

Comment (MichaelCurrin) [reply resources-it.md:14]: I like double spaces before `##` to make markdown more readable.

**Comment by alessiobenvenuti** [comment]
Ok, @MichaelCurrin , now it should be fine, however, let me know if there are other problems. 
Also, let me ask a question that has nothing to do with this PR.  There are news of PR #27?

**Comment by MichaelCurrin** [comment]
Both merged: )

**Pull request merged**